base/bucket_gocb.go

```diff
@@ -1334,13 +1333,31 @@ func (bucket *CouchbaseBucketGoCB) GetWithXattr(k string, xattrKey string, rv in
 				return false, gocb.ErrKeyNotFound, uint64(0)
 			}
 			cas = uint64(res.Cas())
-			return false, nil, cas
 
 		default:
 			shouldRetry = isRecoverableReadError(lookupErr)
 			return shouldRetry, lookupErr, uint64(0)
 		}
 
+		if userXattrKey != "" {
```

Comment:
If we hit the ErrSubDocMultiPathFailureDeleted case above, can we skip attempted user xattr lookup?

Reply:
Yes. Will add